custom/icds_reports/views.py

```diff
@@ -533,6 +533,10 @@ def get(self, request, *args, **kwargs):
         config.update(get_location_filter(location, domain))
         loc_level = get_location_level(config.get('aggregation_level'))
 
+        # Changing the aggregation level to the original convention which is 1 for state,
```

Comment:
Can you make a #TODO comment here to make this change more reusable to other places this works well.